styles.css

```diff
@@ -0,0 +1,96 @@
+body {
+  width: 310px;
+  height: 718px;
+}
+
+.nav-bars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+  width: 340px;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+  height: 580px;
+  padding-bottom: 50px;
+  padding-top: 50px;
+}
+
+.myself {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./photo/bg/header-shapes-mobile.png);
+  padding-top: 20px;
+  padding-right: 24px;
+  width: 400px;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top;
+}
+
+.myself > .greeting {
+  align-items: center;
+  width: 200px;
+  height: 112.75px;
+  color: #172b4d;
+  padding-left: 23px;
+  padding-top: 30px;
+  padding-bottom: 20px;
+}
+
+.logo-show {
+  padding-right: 24px;
+  color: blue;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  padding-left: 24px;
+  padding-top: 14px;
+  padding-bottom: 14px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 10px;
+  line-height: 20px;
+}
```

Comment:
Great job @osbentech 🤝

- [ ] According to project requirements, please double check your button interactions of hover and enable. Please check out the Figma design. The buttons section will guide you. Make sure the hover effect on the logo is fixed.
![buttons](https://user-images.githubusercontent.com/63917725/177218115-fe0c9b47-9521-4f24-be97-9c9ac8112650.PNG)